src/views/RecIM/views/Activity/components/ScheduleList/index.jsx

```diff
@@ -20,23 +29,35 @@ import { windowBreakWidths } from 'theme';
 const ScheduleList = ({ isAdmin, series, activityID, reload, setReload, activityTeams }) => {
   const [anchorEl, setAnchorEl] = useState();
   const [width] = useWindowSize();
-  const openMenu = Boolean(anchorEl);
+  const [showAdminTools, setShowAdminTools] = useState(false);
+  const [showDetailsMenu, setShowDetailsMenu] = useState(false);
   const [openAutoSchedulerDisclaimer, setOpenAutoSchedulerDisclaimer] = useState(false);
   const [openDeleteDisclaimer, setOpenDeleteDisclaimer] = useState(false);
   const [disclaimerContent, setDisclaimerContent] = useState('');
   const [openEditSeriesForm, setOpenEditSeriesForm] = useState(false);
   const [openSeriesScheduleForm, setOpenSeriesScheduleForm] = useState(false);
   const [openMatchForm, setOpenMatchForm] = useState(false);
   const [isMobileView, setIsMobileView] = useState(false);
+  const [seriesSchedule, setSeriesSchedule] = useState();
 
   useEffect(() => {
     if (width < windowBreakWidths.breakSM) setIsMobileView(true);
     else setIsMobileView(false);
   }, [width]);
 
+  useEffect(() => {
+    const loadSchedule = async () => {
+      let fetchedSchedule = await getSeriesSchedule(series.ID);
+      if (fetchedSchedule.ID !== 0) setSeriesSchedule(fetchedSchedule);
+    };
+    loadSchedule();
+  }, [series]);
+
   // default closure
   const handleClose = () => {
```

Comment:
This function name should be more descriptive. We need to know what closing is being handled.